mmdet/models/anchor_heads/fcos_head.py

```diff
@@ -49,7 +49,8 @@ def __init__(self,
                      use_sigmoid=True,
                      loss_weight=1.0),
                  conv_cfg=None,
-                 norm_cfg=dict(type='GN', num_groups=32, requires_grad=True)):
+                 norm_cfg=dict(type='GN', num_groups=32, requires_grad=True),
+                 center_sample_radius=0):  # 0 for disable `center sampling`
```

Comment:
This is somewhat misleading. It may be better to add a parameter `center_sampling` to indicate whether or not using center sampling.